### PR TITLE
Added clone deployment from backup to the commands.

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

File: pkg/format/backup_list.go [hunk 1]
```diff
@@ -38,10 +38,10 @@ func BackupList(list []*backup.Backup, opts Options) string {
 			{"description", x.Description},
 			{"name", x.Name},
 			{"upload", x.Upload},
-			{"url", x.Url},
-			{"state", x.Status.State},
-			{"db-servers", x.DeploymentInfo.Servers.Dbservers},
-			{"uploaded", x.Status.GetUploadStatus().GetUploaded()},
+			{"url", x.GetUrl()},
+			{"state", x.GetStatus().GetState()},
```
Comment (Skarlso) [inline pkg/format/backup_list.go:42]: Status.State blew up with nil pointer when I was trying to list something which had no state yet. Hence, this fix here.

File: cmd/data/clone_deployment_from_backup.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+func init() {
+	cmd.InitCommand(
+		cmd.CloneDeploymentCmd,
+		&cobra.Command{
+			Use:   "backup",
+			Short: "Clone a deployment from a backup.",
+		},
+		func(c *cobra.Command, f *flag.FlagSet) {
+			cargs := &struct {
+				backupID string
```
Comment (Robert-Stam) [inline cmd/data/clone_deployment_from_backup.go:46]: add optional regionID

File: cmd/data/clone_deployment_from_backup.go [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+				conn := cmd.MustDialAPI()
+				ctx := cmd.ContextWithToken()
+				repl := replication.NewReplicationServiceClient(conn)
+				platformc := platform.NewPlatformServiceClient(conn)
+				backupc := backup.NewBackupServiceClient(conn)
+
+				// Check if backup exists
+				if _, err := backupc.GetBackup(ctx, &common.IDOptions{Id: backupID}); err != nil {
+					log.Fatal().Err(err).Str("backup-id", backupID).Msg("Failed to fetch backup")
+				}
```
Comment (Robert-Stam) [inline cmd/data/clone_deployment_from_backup.go:71]: this validation is done inside created already, here you need (additional) permissions

File: cmd/data/clone_deployment_from_backup.go [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+				req := &replication.CloneDeploymentFromBackupRequest{
+					BackupId: backupID,
+				}
+				if cargs.regionID != "" {
+					if _, err := platformc.GetRegion(ctx, &common.IDOptions{Id: cargs.regionID}); err != nil {
+						log.Fatal().Err(err).Str("region-id", cargs.regionID).Msg("Failed to get region.")
+					}
```
Comment (Robert-Stam) [inline cmd/data/clone_deployment_from_backup.go:79]: this validation is done inside created already, here you need (additional) permissions

**Pull request merged**